dockstore-client/src/main/java/io/dockstore/common/FileProvisioning.java

```diff
@@ -136,6 +147,10 @@ private static boolean isCacheOn(INIConfiguration config) {
         return "true".equalsIgnoreCase(useCache) || "use".equalsIgnoreCase(useCache) || "T".equalsIgnoreCase(useCache);
     }
 
+    static String findPluginName(String classObj) {
```

Comment:
As someone who has not dealt with any plugins, I am not sure what the split is for. Please add a comment explaining the purpose of the split.